app/code/Chandan/SalesOrderMessageCustomerAccount/view/frontend/templates/account/dashboard/info.phtml

```diff
@@ -6,6 +6,31 @@
 
 /** @var \Magento\Customer\Block\Account\Dashboard\Info $block */
 ?>
+<?php $_orders = $block->getOrders();?>
```

Comment:
Write this logic in block class

Reply:
I have written this logic in block file. Please check it